### ISSUE TITLE
examples link returns a 404

### ISSUE THREAD

**Issue by mg1075** [Knockout-Contrib/KoGrid#232]
Is the link to the examples page now broken? 
http://ericmbarnard.github.com/KoGrid/#/examples

**Comment by sanderd** [comment]
Same here.
This seems to be the correct link: http://knockout-contrib.github.io/KoGrid/#/examples

I suppose it wasn't updated after the move from Eric's account to Knockout-Contrib.

**Comment by sanderd** [comment]
I updated the link on the wiki page.

**Comment by mg1075** [comment]
The link in the README still needs to be updated.

**Comment by bartburkhardt** [comment]
yes http://ericmbarnard.github.com/KoGrid/#/examples  still returns 404

**Comment by sanderd** [comment]
See issue #246 where the KoGrid-team stated that KoGrid is no longer under (active) development.
Still, the documentation is vital and I'd strongly suggest them to at least update the README-file.

After that, it's a matter of deciding if anyone is willing to take over the project.
(and for that matter, I'd try to participate, but I cannot carry+lead this project)

**Comment by bartburkhardt** [comment]
The Server-Side page example is not a server side example. It only uses a large json file. I could not find info on implementing server side paging.

**Comment by SubashNethaji** [comment]
Is there anyone who can telll me how to get the dropdown value in  Kogrid cell template . Using selected items only it returns data item binded to other columns except . For me I need to bind dropdown from an separate array . I could not get value of it from the dropdown which is in cell template of other grid

**Comment by sanderd** [comment]
@SubashNethaji this is not the place for those kinds of questions.
Not this (TOTALLY unrelated) issue, not the issue tracker as a whole, probably not even github. Try stackoverflow.
